avm/res/insights/data-collection-rule/tests/e2e/direct/main.test.bicep

```
@@ -0,0 +1,105 @@
+targetScope = 'subscription'
+
+metadata name = 'Send data to Azure Monitor Logs with Logs ingestion API.'
```

Minor but suggesting removing the final dot for consistency with other test files
```suggestion
metadata name = 'Send data to Azure Monitor Logs with Logs ingestion API'
```

---

Thanks @eriqua, updated and re-tested